Functions-request-config.js

```diff
@@ -1,46 +1,28 @@
 const fs = require("fs")
+const { Location, ReturnType, CodeLanguage } = require("@chainlink/functions-toolkit")
 
 // Loads environment variables from .env.enc file (if it exists)
 require("@chainlink/env-enc").config()
 
-const Location = {
-  Inline: 0,
-  Remote: 1,
-}
-
-const CodeLanguage = {
-  JavaScript: 0,
-}
-
-const ReturnType = {
-  uint: "uint256",
-  uint256: "uint256",
-  int: "int256",
-  int256: "int256",
-  string: "string",
-  bytes: "Buffer",
-  Buffer: "Buffer",
-}
-
 // Configure the request by setting the fields below
 const requestConfig = {
   // Location of source code (only Inline is currently supported)
   codeLocation: Location.Inline,
+  // Location of secrets (only Remote or DONHosted is supported)
+  secretsLocation: Location.DONHosted,
   // Code language (only JavaScript is currently supported)
   codeLanguage: CodeLanguage.JavaScript,
   // String containing the source code to be executed
   source: fs.readFileSync("./calculation-example.js").toString(),
```

Comment:
@KuphJr 
Wondering whether we need two examples.  The API example will convey all the necessary mental models - and a couple of comments there can help with communication that calcs without HTTP requests are feasible.

However, having to comment in and out parts of the request config is a high-friction step that can be easily overlooked and cause irritating re-runs of the CLI commands.  

This shortest path experience that builds mental models linearly can be achieved with just the API example.

Can we discuss?

Reply:
@zeuslawyer Once you finish this PR, we will have to rebase the [tutorials](https://github.com/smartcontractkit/functions-hardhat-starter-kit/tree/tutorials) branch.
You can see that we only use what's necessary for each example. For instance, this is the [config](https://github.com/smartcontractkit/functions-hardhat-starter-kit/blob/tutorials/tutorials/1-simple-computation/config.js) for the simple computation tutorial.
So you can just run any custom config using `--configpath tutorials/1-simple-computation/config. For instance:

```
npx hardhat functions-request --subid REPLACE_SUBSCRIPTION_ID --contract REPLACE_CONSUMER_CONTRACT_ADDRESS --network REPLACE_NETWORK --configpath REPLACE_CONFIG_PATH
```

> @KuphJr @aelmanaa Wondering whether we need two examples. The API example will convey all the necessary mental models - and a couple of comments there can help with communication that calcs without HTTP requests are feasible.
> 
> However, having to comment in and out parts of the request config is a high-friction step that can be easily overlooked and cause irritating re-runs of the CLI commands.
> 
> This shortest path experience that builds mental models linearly can be achieved with just the API example.
> 
> **This would also reduce the amount of documentation people have to wade through**....but conscious that @aelmanaa may be burdened with updating the docs if we remove the calculation example.
> 
> Can we discuss?

@zeuslawyer Once you finish this PR, we will have to rebase the [tutorials](https://github.com/smartcontractkit/functions-hardhat-starter-kit/tree/tutorials) branch.
You can see that we only use what's necessary for each example. For instance, this is the [config](https://github.com/smartcontractkit/functions-hardhat-starter-kit/blob/tutorials/tutorials/1-simple-computation/config.js) for the simple computation tutorial.
So you can just run any custom config using `--configpath tutorials/1-simple-computation/config. For instance:

```
npx hardhat functions-request --subid REPLACE_SUBSCRIPTION_ID --contract REPLACE_CONSUMER_CONTRACT_ADDRESS --network REPLACE_NETWORK --configpath REPLACE_CONFIG_PATH
```

Reply:
thanks @aelmanaa  - that makes sense since its on the /tutorials branch.  But even on that branch there is the [Functions-request-config.js](https://github.com/smartcontractkit/functions-hardhat-starter-kit/blob/tutorials/Functions-request-config.js) file which has all the commenting in and out of source files.  

And the /tutorials folder has a different source for each example. Which means in the root we really dont need ton have both the simple example and the API example.  

It adds overhead and complexity IMO!

Reply:
If you think we should simplify and remove one of the example source code files, I am fine with that.  We can address in a separate PR.